src/index.d.test.ts

```diff
@@ -0,0 +1,24 @@
+// tslint:disable no-console
+
+import { Config, createForm, AnyObject } from 'final-form'
+import * as arrayMutators from './index'
+import { Mutators } from './index'
+
+const onSubmit: Config['onSubmit'] = (values, callback) => {}
+
+const form = createForm({
+  mutators: { ...arrayMutators },
+  onSubmit
+})
+
+// Get form.mutators (default as object) and cast to Mutators
+const mutators: Mutators = ((form.mutators || {}) as any) as Mutators
```

Comment:
FYI - This casting is a bit gross, but unless we allow parameterized types in the form and expose it all the way through, this is the way it needs to be done.  Parameterized types in jsx is already painful so this is not any worse by any means.